data/backstage/gospel/2025-10-15-gospel.md

```diff
@@ -0,0 +1,54 @@
+
+**[Ortac](https://github.com/ocaml-gospel/ortac)** (released, ready for production codebases) provides dynamic verification through runtime assertion checking and automated test generation. Ortac/QCheck-STM generates black-box state-machine tests, while Ortac/Wrapper instruments functions with runtime assertions. Ortac/Monolith provides fuzzing integration and remains experimental. These tools have already found real bugs including missing bounds checks in [varray](https://github.com/art-w/varray), integer overflows causing segfaults in [bitv](https://github.com/backtracking/bitv), inconsistent behavior with zero-length vectors in bitv, and unexpected behavior in the standard library's Hashtbl.create.
+
+**[Cameleer](https://github.com/ocaml-gospel/cameleer)** (working toward first release) translates Gospel into [Why3](https://why3.lri.fr/) for deductive verification using automated theorem provers.
+
+**[Peter](https://github.com/ocaml-gospel/peter)** (ongoing research) explores separation logic verification through [CFML](https://gitlab.inria.fr/charguer/cfml) integration with [Coq](https://coq.inria.fr/) for complex heap manipulation proofs.
+
+**[Gospel language](https://github.com/ocaml-gospel/gospel)** (experimental, stabilizing) continues to evolve with ongoing design decisions, but the core concepts are established and the type-checker ensures specifications stay synchronized with code.
+
+## Open Source Infrastructure Strategy
+
+Gospel development is funded by [ANR grant ANR-22-CE48-0013](https://anr.fr/Project-ANR-22-CE48-0013), executed through collaboration between [Nomadic Labs](https://www.nomadic-labs.com/), [Tarides](https://tarides.com/), [Inria](https://www.inria.fr/), and [LMF (Laboratoire Méthodes Formelles)](https://lmf.cnrs.fr/) at Université Paris-Saclay. The project demonstrates how open source can transform formal verification economics: rather than each organization independently funding expensive verification efforts, shared open tooling and verified libraries benefit the entire ecosystem.
+
+## Current State and Platform Integration
+
+Ortac/QCheck-STM and Ortac/Wrapper are released and have proven effective on real codebases. Test runs complete in hundreds of milliseconds, making them practical for CI pipelines. Gospel itself remains experimental with language design still evolving, but it's mature enough for early adopters to write specifications and provide valuable feedback.
+
+The [OCaml Platform roadmap](https://ocaml.org/tools/platform-roadmap#w22-formal-verification) through 2026 includes formal verification as a first-class workflow. The vision: when specifications are defined, `dune test` will automatically report verification failures alongside regular test results, with failures surfaced in editors through the LSP server. [Dune](https://dune.build/) will orchestrate verification tools transparently, making formal methods as accessible as formatting or linting.
```

Comment:
One still have to write dune rules to run the verification tool. When I asked for integrating Ortac or Gospel to Dune I received a strong **no** the then dune team (in Tarides) and I don't believe this is going to change.